website/src/components/Layout/ChatLayout.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Flex } from "@chakra-ui/react";
+import { BarChart2, ExternalLink, Layout, MessageCircle, MessageSquare, TrendingUp } from "lucide-react";
+import { getEnv } from "src/lib/browserEnv";
+
+import { ChatListDesktop } from "../Chat/ChatListDesktop";
+import { ChatListMobile } from "../Chat/ChatListMobile";
+import { SlimFooter } from "../Dashboard/SlimFooter";
+import { HeaderLayout } from "../Header/Header";
+import { SideMenuLayout } from "../SideMenuLayout";
+import { ToSWrapper } from "../ToSWrapper";
+
+export const getChatLayout = (page: React.ReactElement) => (
+  <HeaderLayout preLogoSlot={<ChatListMobile />}>
+    <ToSWrapper>
+      <SideMenuLayout
+        collapsed
+        items={[
```

Comment:
is this the same list used in `DashboardLayout`? if so, can we extract it to some function?